### ISSUE TITLE
Make emojis screen-reader friendly

### ISSUE THREAD

**Issue by ruthmoog** [ruthmoog/portfolio#7]
Check screen reader output for text which includes emojis.

At the moment text such as, "☕️ The main coding language I use for my job is Java.", is read by screen-readers as, "hot beverage The main coding language I use for my job is Java.".

Screen readers should pass over emojis which are used for presentation to provide visual context, but are not helpful to the text content.

It should be possible to use the span element to improve screen-reader output where relevant.

More info here: [litmus.com: Special Characters, Emojis, Line Breaks: More Tricks for Optimizing Your Emails for Screen Readers](https://www.litmus.com/blog/special-characters-emojis-line-breaks-more-tricks-for-optimizing-your-emails-for-screen-readers/)

**Comment by ruthmoog** [comment]
emoji are wrapped with span element, with ARIA hidden: 
```html
<span class="emoji" aria-hidden="true">🦥 </span>
```

and speak is set in CSS:

```CSS
.emoji {
    speak: none;
}
```
